dissect/target/helpers/configutil.py

```diff
@@ -281,6 +385,15 @@ def _select_parser(entry: FilesystemEntry, hint: Optional[str] = None) -> Parser
     if hint and (parser_type := CONFIG_MAP.get(hint)):
         return parser_type
 
+    matches = []
+
+    for match, value in MATCH_MAP.items():
+        if fnmatch(entry.path, f"*{match}"):
```

Comment:
I would add the`*` in `f"*{match}"` to the entries in `MATCH_MAP` itself, as that would make it clear there that it will map the subdir/file at any position in the filesystem.